### PR TITLE
Added fish shell completion

### PR DESCRIPTION
**Pull request by mhdali**
Do I need to update the steps as well in the following file?

https://raw.githubusercontent.com/exercism/exercism.io/31557745b400d6b2c9ad6945a76092bdde43bd61/x/docs/md/cli/completion.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,14 @@ The webapp is set up with a Procfile. It simply requires `foreman` or `forego` t
 1. `bundle install`
 1. `foreman start`
 
+## Docker env
```
Comment (Insti) [inline README.md]: Is this related to the fish shell completion?

Comment (mhdali) [reply README.md]: Not really, but because I had to run the env on Docker, so I thought I could update the Readme file for that.

Comment (Insti) [reply README.md]: Ok. Updating the Readme is helpful too :)
It is possibly worth making it a separate pull request.

Comment (kytrinyx) [reply README.md]: Thanks @Insti.

@mhdali the docker change is completely separate, and while I'd like to accept the changes that relate to fish, I'm not so sure about the Docker instructions and would prefer to discuss those separately.

**Review by kytrinyx — CHANGES_REQUESTED** [review]
The fish-related changes look good. Would you mind removing the unrelated changes so we can get this merged and deployed? 🌷

**Comment by kytrinyx** [comment]
@mhdali Would you have time to finish this one up?
